examples/basic/command_classes.py

```diff
@@ -10,8 +10,8 @@
 
 
 @bot.include
-@crescent.command(name="echo")
-class Say:
+@crescent.command
+class say:
```

Comment:
I mean... I guess this works. I suppose I can just use the name= option.

Reply:
I made this work in the PR. I should've made that more clear.